### DISCUSSION TITLE
Server disconnected errors while uploading with prepdocs.sh

### DISCUSSION THREAD

**Discussion by HughRunyan** [microsoft/AI-Chat-App-Hack#46] [General]
If anyone has had disconnected/timeout type problems while uploading their documents and found ways around it, I'd love to know! The problem (mine at least) is coming from line 81 in pdfparser.py

poller = await form_recognizer_client.begin_analyze_document(model_id=self.model_id, document=content)

**Comment by pamelafox** [comment]
Looks like we have a thread tracking that issue here:
https://github.com/Azure-Samples/azure-search-openai-demo/issues/485

I'll ping the SDK team to see if they have any suggestions.

**Comment by HughRunyan** [comment]
Thanks!

**Reply by pamelafox** [reply]
@HughRunyan Are you able to share the PDF that caused the timeout? The SDK team is trying to replicate. (Don't share if it's private of course).
